docs/display_panels.rst

```diff
@@ -2,147 +2,103 @@
-
-Once the two dimensional data element is determined, the data is converted to scalar, if required, according to the complex data type in the inspector. For instance, if the data is complex then it may be converted to a scalar by taking the log-modulus of the data.
-
-Once the data has been converted to scalar values, brightness, contrast, and gamma adjustments are applied to rescale the data.
-
-The display has an associated color map, which can be changed in the Inspector.
-
-Once the adjustments have been applied, the data is scaled so that the lower display limit maps to the lower end of the color map and the upper display limit maps to the upper end of the color map.
-
-The display limits may be specified or unspecified, in which case they are "auto" calculated, meaning that they will automatically adjust to the lowest and highest values of the data (once it has been converted to scalar and brightness, contrast, and gamma applied) in the image.
-
-The selection of indexes for sequences or collections, the conversion from complex to scalar, the adjustments such as brightness, contrast, and gamma, the display limits, and the color map are all selectable or editable using the inspector (see :ref:`Inspector Panel`).
-
-You can press the :kbd:`Enter` key to reset display limits.
-
-The Histogram panel (see :ref:`Histogram Panel`) shows the histogram of the image data. It can be used to set or reset the display limits by dragging within the histogram graph or by double clicking within the graph, respectively.
-
-The Info panel (see :ref:`Info Panel`) shows the value of the image data underneath the cursor.
-
-You can add graphics and other annotations to the image display. See :ref:`graphics`.
+======
+An image is the visualization of 2D data in a display panel. There are several ways to adjust the view of an image.
+
+.. image:: graphics/images_infographic.png
+    :width: 792
+    :alt: Adjusting Images
+
+* Move an image by selecting the move tool from the toolbar and then clicking and dragging on the image. If you are using a trackpad, you can also move the image by dragging with two fingers while the cursor is on top of the image.
+  
+* Scale images by selecting one of the scale options from the toolbar or zoom in and out of an image by using the + and - keys. A data item will be set to fit to the enclosed space by default when it is first displayed. With the toolbar buttons, you can choose to have the image fit to the display panel, fill the display panel, scale to a 1:1 ratio of image pixels to screen pixels, or scale to a 2:1 ratio of image pixels to screen pixels. If the scale is set to fit or fill the display panel, the image will rescale with the display panel if it is resized. If the scale is set to either the 1:1 or 2:1 ratios, the image will not scale with the display panel. Scaling of a selected image can also be performed by using the Display menu and selecting Fit to View, Fill View, 1:1 View, or 1:2 View.
```

Comment:
This is very dense information content for a bullet point. Maybe break this out into its own section?

One rule of thumb is that bullet points should make it easier for the reader. The use of bullet points here is odd since the information is very dense and, if anything, the bullets make it more difficult for the reader.